execution/executor-types/src/lib.rs

```diff
@@ -153,7 +153,7 @@ impl StateComputeResult {
 
 impl StateComputeResult {
     pub fn version(&self) -> Version {
-        max(self.num_leaves, 1) - 1
+        max(self.num_leaves, 1).saturating_sub(1)
```

Comment:
I think this is worse than panicking...

Reply:
you're probably right, should I then do something like "max(self.num_leaves, 1).checked_sub(1).expect("overflow occurred") " ?

Reply:
I like that @xvschneider  :D Thank you.